.github/actions/is-release-week-v1/action.yml

```
@@ -0,0 +1,12 @@
+name: 'Check Release Week'
+description: 'Determine if release should continue to run or stop'
+inputs:
+  oddWeek:
+    description: 'True if the action should run on odd weeks, False if the action should run on even'
+    required: true
```

We can default the value to `false` here: 

```suggestion
  oddWeek:
    description: 'True if the action should run on odd weeks, False if the action should run on even'
    default: 'false'
    required: true
```